template.hbr

```diff
@@ -224,6 +230,11 @@ players:
                     "font_style": null,
                     "font_weight": null
                 },
+                "module": {
+                    "color": "#{{lavender}}",
+                    "font_style": "italic",
+                    "font_weight": null
+                },
```

Comment:
We've moved away from lavender while defining the newer style guide, I'd be curious how VSCode highlights modules - I'd imagine its yellow?

Reply:
it is indeed yellow! 
<img width="511" alt="image" src="https://github.com/catppuccin/zed/assets/21319730/1733a6e3-3430-4da9-95a5-c85dc975d070">